src/routes/courseRoutes.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import { createCourse, updateCourse, deleteCourse, getCourse } from '../controllers/courseController';
+
+const router = Router();
+
+router.post('/courses', createCourse);
+
+router.put('/courses/:id', updateCourse);
+
+router.delete('/courses/:id', deleteCourse);
+
+router.get('/courses/:id', getCourse);
```

Comment:
Please create validators and use them in the routes along with the validate middleware.

Refer to the auth routes to check it out.